src/components/popover/popover_footer.tsx

```diff
@@ -39,6 +39,7 @@ export const EuiPopoverFooter: EuiPopoverFooterProps = ({
 }) => {
   const classes = classNames(
     'euiPopoverFooter',
+    // @ts-expect-error EuiPanel increased its available sizes
```

Comment:
Just to make sure I'm understanding these changes correctly, what's the plan for eventually removing these `@ts-expect-error` comments?

Reply:
When these components get converted they will be able to inherit from the global padding style too, but right now it's all manual class names and Sass maps.

Reply:
Oh actually, I think what might be a better solution is to change the name of the originally exported `PADDING_SIZES` from EuiPanel, into a generic file and call it `PADDING_SIZES_DEPRECATED` that way it won't fight the values too

Reply:
Ah OK, I think I see it in above comment...

```
// Exported padding sizes and class names necessary for EuiPopover and EuiCard.
// Which currently will only maintain support for the original values until conversion.
```

Since you may not necessarily be the dev that's converting these components, I'd encourage adding a more explicit TODO in this comment as to when we would 'fix' this.

```suggestion
    // @ts-expect-error EuiPanel increased its available sizes. When we convert this component, we should also increase size support to match EuiPanel and remove this comment
```

Reply:
Oops sorry, GitHub was being stale and not showing me your comments before I made my 2nd one. Either way works!